src/app/pages/ParaTimesPage/TransactionRecipient/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -52,21 +35,16 @@ exports[`<TransactionRecipient /> should render EVMc withdraw variant component
 
 }
 
-<div
+<span
   class="c0"
   data-testid="paraTime-content-description"
-  style="max-width: 550px;"
 >
-  <span
-    class="c1"
-  >
-    Please enter the address of the withdrawing wallet on the 
-    <strong>
-      Emerald
-    </strong>
-     (EVMc) ParaTime and then click "Next"
-  </span>
-</div>
+  Please enter the address of the withdrawing wallet on the 
+  <strong>
+    Emerald
+  </strong>
+   (EVMc) ParaTime and then click "Next"
```

Comment:
Text is misleading

- When withdrawing from Emerald user should enter **withdrawing private key** and **receiving address**
![image](https://user-images.githubusercontent.com/3758846/189253611-73e9611c-b667-4bcd-8dbd-e0238505f451.png)

- When withdrawing from Cipher user enters **receiving address**
![image](https://user-images.githubusercontent.com/3758846/189253716-87e42854-c84a-48e8-86a3-675b576915ad.png)